week6/test_bank/test_bank.py

```diff
@@ -0,0 +1,29 @@
+import bank
+
+def test_20_dollor():
+    result = "$20"
+    assert bank.check("How you doing?") == result
+
+    result = "$20"
```

Comment:
you don't need to repeat the assignment, `result = "$20"` is in scope for all the assert in the test function.  Same for the other test functions

Reply:
just made the change about to push it with week7 tomorrow